### PR TITLE
add configure_gimbal to support AC3.3

### PR DESCRIPTION
**Pull request by squilter**
The gimbal control that was added yesterday works fine with AC3.2.  Just set the parameter MNT_MODE to 2, then use `GimbalApi.setGimbalOrientation()`.

Starting with AC3.3, that parameter does not exist.  Instead, use this new function, `GimbalApi.configureGimbal()`, before sending the `GimbalApi.setGimbalOrientation()`.

### CODE REVIEW

**Comment by ne0fhyk** [comment]
@squilter  let's automate that process. The user should be using the same api regardless of the version of ardupilot he is talking to.
In this case, the drone object can be used to retrieve the version of the autopilot. Then the logic should use the path appropriate to that version.

**Comment by squilter** [comment]
What's the best way to see if the parameter `MNT_MODE` exists?

**Comment by squilter** [comment]
@ne0fhyk Okay, now it supports old arducopter version also.  I tested this method on AC3.2 and 3.3.

File: dependencyLibs/Core/src/org/droidplanner/core/MAVLink/command/doCmd/MavLinkDoCmds.java [hunk 1]
```diff
@@ -99,6 +103,40 @@ public static void setServo(Drone drone, int channel, int pwm) {
...
+            msg.target_system = drone.getSysid();
+            msg.target_component = drone.getCompid();
+            msg.mount_mode = (byte) mountMode;
+            msg.stab_pitch = stabilizePitch ? (byte) 1  : 0;
+            msg.stab_roll = stabilizeRoll ? (byte) 1 : 0;
+            msg.stab_yaw = stabilizeYaw ? (byte) 1 : 0;
+            drone.getMavClient().sendMavPacket(msg.pack());
+        }else{
+            drone.getParameters().sendParameter("MNT_MODE", 1, mountMode);
+            mountParam.value = mountMode;
```
Comment (squilter) [inline dependencyLibs/Core/src/org/droidplanner/core/MAVLink/command/doCmd/MavLinkDoCmds.java]: This is the only part that I'm not sure about.  I needed to send a new parameter.  This was the best I came up with.  Is there a better way?

**Comment by squilter** [comment]
@ne0fhyk What should I do with this PR now, was something wrong with it the other day?

**Comment by squilter** [comment]
replaced by #196

**Pull request closed without merging**